controller.go

```diff
@@ -51,9 +54,16 @@ func (c *OFController) sendEchoLoop() {
 	// send echo request forever
 }
 
-func ServerLoop() {
-	serverStr := ":6633"
-	tcpAddr, err := net.ResolveTCPAddr("tcp", serverStr)
+func ServerLoop(listenPort int) {
+	var port int
+
+	if listenPort <= 0 {
```

Comment:
The listenPort should be checked to confirm  it is NOT larger than upper-bound of port number looks like bellow.
```
if listenPort <=0 || listenPort >= 65536 {
...
```